inst/htmlwidgets/datatables.js

```diff
@@ -8,8 +8,9 @@
 var DTWidget = {};
 
 // 123456666.7890 -> 123,456,666.7890
-var markInterval = function(d, digits, interval, mark, decMark, precision) {
+var markInterval = function(d, digits, interval, mark, decMark, precision, exponential) {
   x = precision ? d.toPrecision(digits) : d.toFixed(digits);
+  if(exponential) {x = exponential ? d.toExponential(digits) : d.toFixed(digits);};
```

Comment:
Here `d.toFixed(digits)` will never be executed no matter if `exponential` is true or false.